### ISSUE TITLE
Generated component implementations are not recognized during project build.

### ISSUE THREAD

**Issue by MarinKacaj** [google/dagger#281]
When I add new dependencies and providers, I get "symbol not found DaggerMyComponent" error messages during build, even though DaggerMyComponent is generated correctly and currently resides in the appropriate package/directory. Android Studio too acknowledges its existence (I press Ctrl + B over DaggerMyComponent and the relevant file opens up in the editor), but the build process doesn't.

**Comment by digitalbuddha** [comment]
Let's get the basics out of the way.  Version of Studio, Gradle? Does this happen every time? How do you get past it/when did it start? Do you use APT plugin? what folder is the generated file in?  Using jRebel or Live Reload?

**Comment by MarinKacaj** [comment]
- Versions:
Android Studio 1.5.1
Gradle 2.8
- Following the same steps as last time I encountered the problem, it didn't happen again.
- I temporarily remove all code related to the generated DaggerMyComponent and then add it again. (I've just started developing the app today, so it's not much trouble, but it might become a problem with scaling).
- It started the first time I added new dependencies.
- Yes, I use the apt plugin.
- The generated file is in: <project name>/<module name>/build/generated/source/apt/debug/<my reversed domain>/internal/di/components/DaggerMyComponent.
- For the moment I'm not using jRebel or Live Reload.
